src/SFA.DAS.EmployerFinance.MessageHandlers/CommandHandlers/ImportPayeSchemeLevyDeclarationsCommandHandler.cs

```diff
@@ -0,0 +1,22 @@
+using System.Threading.Tasks;
+using MediatR;
+using NServiceBus;
+using SFA.DAS.EmployerFinance.Application.Commands.ImportPayeSchemeLevyDeclarations;
+
+namespace SFA.DAS.EmployerFinance.MessageHandlers.CommandHandlers
+{
+    public class ImportPayeSchemeLevyDeclarationsCommandHandler : IHandleMessages<ImportPayeSchemeLevyDeclarationsCommand>
```

Comment:
I know these command handlers are pretty lightweight as-is, but what do you think of introducing a generic base class for them?

Reply:
There might be something smart we could do with NServiceBus middleware that automatically forwards them to Mediatr handlers. Let's leave them for now but see what we can do when we introduce the payments workflow.

Reply:
That sounds like a good idea!